### PR TITLE
timer update

### PR DESCRIPTION
**Pull request by rStrzelczyk98**

### CODE REVIEW

File: src/app/timer-card/timer-card.component.ts [hunk 1]
```diff
@@ -19,70 +10,76 @@ export type Status = {
...
       })
     );
   }
 
-  private alarm() {
-    new Audio(
-      'http://codeskulptor-demos.commondatastorage.googleapis.com/GalaxyInvaders/pause.wav'
-    ).play();
+  private setStreams() {
+    setTimeout(() => {
```
Comment (RdN) [inline src/app/timer-card/timer-card.component.ts:60]: Why so?

File: src/app/timer-card/timer-card.component.ts [hunk 2]
```diff
@@ -19,70 +10,76 @@ export type Status = {
...
+        .getTimerStatus(this.id)
+        .subscribe(({ active }) => (this.isPaused = active));
+      this.toggleButtonColor();
+      this.countdown$ = this.ts.getTimerStream(this.id);
+      this.progress$ = this.progressBarValue(this.timeInSeconds);
+    });
+  }
+
+  private resetStreams() {
+    setTimeout(() => this.ts.resetTimerStream(this.id));
```
Comment (RdN) [inline src/app/timer-card/timer-card.component.ts:75]: Why in timeout?

**Review by RdN — CHANGES_REQUESTED** [review]

File: src/app/timer-card/timer-card.component.ts (hunk 1 above)
Comment (rStrzelczyk98) [reply src/app/timer-card/timer-card.component.ts:60]: Without timeout, throws Error: "expression changed after it has been checked error"

File: src/app/timer-card/timer-card.component.ts (hunk 2 above)
Comment (rStrzelczyk98) [reply src/app/timer-card/timer-card.component.ts:75]: Without timeout, throws Error: "expression changed after it has been checked error"

File: src/app/timer-card/timer-card.component.ts (hunk 1 above)
Comment (RdN) [reply src/app/timer-card/timer-card.component.ts:60]: Wonder if we can use something else to approach it.

**Review by RdN — APPROVED** [review]

**Pull request merged**